data/dineshlalam15.json

```diff
@@ -17,7 +17,7 @@
       "name": "Hey! Let's connect on LinkedIn",
       "url": "https://www.linkedin.com/in/dinesh-kumar-lalam-2584a8207/",
       "icon": "FaLinkedIn"
```

Comment:
Hi @dineshlalam15, thank you for contributing  :tada:
Please replace it with &nbsp; **' FaLinkedin '** &nbsp; :+1: 
```Suggestion
      "icon": "FaLinkedin"
```